app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+
+  belongs_to :user
+  default_scope -> { order(created_at: :desc) }
```

Comment:
uh anh biết đâu. cái ni bác unit test thích thế mình chiều ak. còn anh thì ko thích set default, khách hoặc sếp to bảo thì làm :v: 
Tốt nhất nếu querry order groupby gì nhiều thì làm 1 cái scope riêng. dùng ông default thì chỉ có nước bị complaint bể đầu :D